modules/ROOT/pages/using-policies.adoc

```diff
@@ -1,13 +1,13 @@
-= Applying a Policy
+= Apply a Policy
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 :keywords: policy, endpoint
 
 
-By default, a policy applies to the entire API. 
+You can apply a policy to the entire API (default) or to specific methods and resources of an API. 
```

Comment:
Just a suggestion: inclusion of the word "either" prepares the audience for a choice:
```suggestion
You can apply a policy either to the entire API (default) or to specific methods and resources of an API. 
```

Reply:
Done! Thanks!